### PR TITLE
Clarify that the parser operates on the entire response

### LINKED ISSUE #2: JSON vs Javascript lists
What is your plan for handling scenarios where the first 1024 bytes look like a beginning of list?

    [ 1, 2, 3, "1024 bytes like that", ....

Such a prefix can be extended to form valid JSON and invalid Javascript:

    [ 1, 2, 3, "foobar" ]

or it can be extended to form valid Javascript and invalid JSON:

    [ 1, 2, 3, "foobar" ].map(x => console.log(x))

I assume that step 14 works with the first 1024 bytes (otherwise we increase further the latency of making the first bytes available in the renderer process).  OTOH, step 14 says "response's body" rather than "bytes", so maybe this is a wrong assumption.

PS. Is there a Mozilla bug we can follow to see if the proposed algorithm works well in practice?  If Javascript sniffing turns out to be robust against real websites, then the proposed algorithm seems like an important improvement in coverage of CORB.

### PR DESCRIPTION
**Pull request by annevk**
Closes #2.

### CODE REVIEW

**Comment by annevk** [comment]
@anforowicz care to review? I added you to the repository, but I cannot add you as reviewer just yet it seems.

File: README.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ To determine whether to allow response _response_ to a request _request_, run th
 1. If _response_'s status is not an [ok status](https://fetch.spec.whatwg.org/#ok-status), then return false.
 1. If _mimeType_'s essence starts with "`audio/`", "`image/`", or "`video/`", then return false.
 1. If _mimeType_'s essence is "`text/csv`", then return false.
+1. Wait for end-of-file of _response_.
```
Comment (anforowicz) [inline README.md:41]: This probably means that the size of the response is leaked via timing side channels (e.g. see also https://crbug.com/1154250).  OTOH, I am not sure what can be done to fix this problem.

Would it be worth adding a note here to explain why we need to parse the whole response / what would go wrong if we parsed only the first 1024 bytes or so?  AFAIR, the problem is that the first 1024 bytes can look like valid Javascript (e.g. unterminated HTML comment, unterminated JSON list) - unconditionally blocking such responses may block valid Javascript;  unconditionally allowing such responses may disclose cross-site non-script-data.

/cc @csreis

Comment (annevk) [reply README.md:41]: Yeah, that seems right. If your JS parser can do things incrementally you could avoid that sometimes perhaps.

It's also that by parsing 1024 bytes your JS parser might not be in a state that makes sense and there's definitely no specification that can handle that scenario currently.

**Review by anforowicz — APPROVED** [review]
Thanks @annevk - LGTM!

**Pull request merged**